inspire/modules/workflows/tasks/submission.py

```diff
@@ -121,7 +124,7 @@ def _reply_ticket(obj, eng):
         from flask import render_template
 
         ticket_id = obj.extra_data.get("ticket_id", "")
-        if not ticket_id:
+        if not ticket_id or not cfg.get("PRODUCTION_MODE"):
```

Comment:
I believe it should be `not cfg.get("PRODUCTION_MODE")`, because it should only continue if we're in production mode.